twoliter/embedded/rpm2img

```diff
@@ -168,6 +167,7 @@ rpm -iv --ignorearch --root "${ROOT_MOUNT}" "${PACKAGE_DIR}"/*.rpm
 
 # Inventory installed packages.
 INVENTORY_QUERY="\{\"Name\":\"%{NAME}\"\
+,\"SourcePackage\":\"%{SOURCERPM}\"\
```

Comment:
Can you double-check that there are no source package fields in the final application inventory? And also please check that a new node correctly reports inventory into SSM after this change.